website/docs/Examples/Integrate - OpenAI.md

```diff
@@ -0,0 +1,194 @@
+FLAML has integrated the OpenAI's completion API. In this example, we will tune several hyperparameters including the temperature, prompt and n to optimize the inference performance of OpenAI's completion API for a code generation task. Our study shows that tuning hyperparameters can significantly affect the utility of the OpenAI API.
+
+### Prerequisites
+
+Install the [openai] option. The option is available in flaml since version 1.1.3. This feature is subject to change in future versions.
+```bash
+pip install "flaml[openai]==1.1.3"
+```
+
+
+Setup your OpenAI key:
+```python
+import os
+
+if "OPENAI_API_KEY" not in os.environ:
+    os.environ["OPENAI_API_KEY"] = "<your OpenAI API key here>"
+```
+
+If you use Azure OpenAI, uncomment the following:
+
+Setup your OpenAI key:
+```python
+# openai.api_type = "azure"
+# openai.api_base = "https://<your_endpoint>.openai.azure.com/"
+# openai.api_version = "2022-12-01"  # change if necessary
+```
```

Comment:
This is not a notebook, so don't say "uncomment the following" etc. Also, the code snippet is not about "setup key".